projects/swimlane/ngx-ui/src/lib/components/code-editor/code-editor.component.ts

```diff
@@ -61,8 +61,8 @@ const CODEMIRROR_VALUE_ACCESSOR = {
     './foldgutter.css',
     './dracula.css',
     './hint.scss',
-    './code-editor.component.scss'
-  ]
+    './code-editor.component.scss',
+  ],
```

Comment:
Did our prettier config change to include trailing commas?

Reply:
Yes

Reply:
Actually.. not a change in or config but in prettier itself.

Reply:
https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-trailingcomma-to-es5-6963httpsgithubcomprettierprettierpull6963-by-fiskerhttpsgithubcomfisker